pkg/auth/cleanup/service.go

```diff
@@ -76,6 +82,10 @@ func (s *Service) Run(config *v3.AuthConfig) error {
 		return fmt.Errorf("error cleaning up users: %w", err)
 	}
 
+	if err := s.deleteTokens(config); err != nil {
+		return fmt.Errorf("error cleaning up tokens: %w", err)
```

Comment:
Maybe this could include the `AuthConfig` name that it failed to cleanup the tokens for?

Otherwise this is very similar to the error returned by `deleteTokens()` in which case, it's not adding additional context?

Reply:
Yep, the first deletion has this info in the error message, while the others not.

I will add more context to them. :+1: